### PR TITLE
Add specification on top-nav

### PR DESCRIPTION
**Pull request by Dindihub**
This PR covers issue #18 

 Moved specification on top-nav following a recommendation on the [TUF documentation analysis](https://github.com/cncf/techdocs/tree/main/analyses/0012-TUF)

Please review @chalin @lukpueh

### CODE REVIEW

File: content/en/specifications/specification-index.md
Comment (chalin) [inline content/en/specifications/specification-index.md:1]: Drop this page

File: content/en/specifications/specification-latest.md [hunk 1]
```diff
@@ -0,0 +1,29 @@
+---
+title: Specification index
+weight: 15
+description:
+---
+
+The following doc versions are available:
```
Comment (chalin) [inline content/en/specifications/specification-latest.md]:
```suggestion
The following specification versions are available:
```

File: content/en/specifications/specification-latest.md [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+- [v1.0.24](https://theupdateframework.github.io/specification/v1.0.24/index.html)
+- [v1.0.23](https://theupdateframework.github.io/specification/v1.0.23/index.html)
+- [v1.0.22](https://theupdateframework.github.io/specification/v1.0.22/index.html)
+- [v1.0.20](https://theupdateframework.github.io/specification/v1.0.20/index.html)
+- [v1.0.19](https://theupdateframework.github.io/specification/v1.0.19/index.html)
+- [v1.0.18](https://theupdateframework.github.io/specification/v1.0.18/index.html)
+- [v1.0.17](https://theupdateframework.github.io/specification/v1.0.17/index.html)
+
+Find here an updated list of the
+[spec versions](https://theupdateframework.github.io/specification/)
```
Comment (chalin) [inline content/en/specifications/specification-latest.md]: Eventually we might not need the link to the GH pages site, but for now I'd like to keep it. Let's adjust the prose though:

```suggestion
This list is also available from
[Spec versions](https://theupdateframework.github.io/specification/).
```

**Review by chalin — COMMENTED** [review]
Now that I'm seeing this, I think that we can rework (simplify) this. See inline comments.

**Comment by Dindihub** [comment]
This is ready for another review @chalin

**Review by lukpueh — CHANGES_REQUESTED** [review]
The text reads like a general TUF project overview, and only vaguely prepares the reader for the specification. 

I suggest to remove it and just leave the index. Or add a small paragraph about what the user will find in the spec. Maybe you can take inspiration from [this talk](https://youtu.be/1rM8a9cPo-I?t=1107). More specifically, about how the spec defines a detailed client updater workflow and otherwise TUF repo primitives (roles, metadata, delegations), which can be used as building blocks.

**Comment by Dindihub** [comment]
> The text reads like a general TUF project overview, and only vaguely prepares the reader for the specification.
> 
> I suggest to remove it and just leave the index. Or add a small paragraph about what the user will find in the spec. Maybe you can take inspiration from [this talk](https://youtu.be/1rM8a9cPo-I?t=1107). More specifically, about how the spec defines a detailed client updater workflow and otherwise TUF repo primitives (roles, metadata, delegations), which can be used as building blocks.

@lukpueh Okay. I'll try. Thanks

**Comment by Dindihub** [comment]
@lukpueh I have edited the section and added in the video you shared for reference. Let me know what you think.

File: content/en/specifications/specification-latest.md
Comment (chalin) [inline content/en/specifications/specification-latest.md:1]: Drop this page. Move the content to the "main" (i.e., only) specification page.

File: content/en/specifications/_index.md
Comment (chalin) [inline content/en/specifications/_index.md:1]: Rename this file to `content/en/specs.md`, which means eliminating the folder, and merging the version list into this new page.

File: content/en/specifications/_index.md [hunk 3]
```diff
@@ -0,0 +1,18 @@
+---
+title: Specification
+linkTitle: Specification
```
Comment (chalin) [inline content/en/specifications/_index.md]: Don't include a linkTitle if it matches the `title`, or consider using:

```suggestion
linkTitle:  Spec
```

File: content/en/specifications/_index.md [hunk 4]
```diff
@@ -0,0 +1,18 @@
...
+menu: { main: { weight: 35 } }
+cascade:
+  type: docs
+---
+
+The TUF spec minimizes key compromise impact by supporting roles with multiple
+keys and threshold trust, while ensuring that highly vulnerable online keys are
+not used for trusted roles. Designed for flexibility, it is easy to integrate
+with various software update systems.
```
Comment (chalin) [inline content/en/specifications/_index.md]: I'm not sure that I can make sense of this paragraph. I'd suggest replacing it quite simply by the list of specification versions:

```suggestion
The following specification versions are available:

- [latest](https://theupdateframework.github.io/specification/latest/index.html)
- [draft](https://theupdateframework.github.io/specification/draft/index.html)
- [v1.0.33](https://theupdateframework.github.io/specification/v1.0.33/index.html)
- [v1.0.32](https://theupdateframework.github.io/specification/v1.0.32/index.html)
- [v1.0.31](https://theupdateframework.github.io/specification/v1.0.31/index.html)
- [v1.0.30](https://theupdateframework.github.io/specification/v1.0.30/index.html)
- [v1.0.29](https://theupdateframework.github.io/specification/v1.0.29/index.html)
- [v1.0.28](https://theupdateframework.github.io/specification/v1.0.28/index.html)
- [v1.0.27](https://theupdateframework.github.io/specification/v1.0.27/index.html)
- [v1.0.26](https://theupdateframework.github.io/specification/v1.0.26/index.html)
- [v1.0.25](https://theupdateframework.github.io/specification/v1.0.25/index.html)
- [v1.0.24](https://theupdateframework.github.io/specification/v1.0.24/index.html)
- [v1.0.23](https://theupdateframework.github.io/specification/v1.0.23/index.html)
- [v1.0.22](https://theupdateframework.github.io/specification/v1.0.22/index.html)
- [v1.0.20](https://theupdateframework.github.io/specification/v1.0.20/index.html)
- [v1.0.19](https://theupdateframework.github.io/specification/v1.0.19/index.html)
- [v1.0.18](https://theupdateframework.github.io/specification/v1.0.18/index.html)
- [v1.0.17](https://theupdateframework.github.io/specification/v1.0.17/index.html)

This list is also available from
[Spec versions](https://theupdateframework.github.io/specification/).
```

File: content/en/specifications/_index.md [hunk 5]
```diff
@@ -0,0 +1,18 @@
...
+
+Learn more about the spec from our latest
+[specification](https://theupdateframework.github.io/specification/latest/) docs
+or watch the following video.
```
Comment (chalin) [inline content/en/specifications/_index.md]:
```suggestion
To learn more, see the latest version of the
[specification](https://theupdateframework.github.io/specification/latest/),
or watch the following video.
```

**Review by chalin — COMMENTED** [review]
@lukpueh and I seem to have the same idea of having a single page (in fact, that is what I tried to convey in my round of comments from earlier).

Please make the changes suggested in the comments of this review.

**Review by lukpueh — APPROVED** [review]

**Review by chalin — APPROVED** [review]
LGTM.

There will be followup improvements to make (e.g., possibly not using `type: docs` or maybe hiding the left nav, but that can be done later.

**Pull request merged**